compiler/passes/errorHandling.cpp

```diff
@@ -128,13 +128,30 @@ try {
 */
 
 
-// Static functions
-static bool canBlockThrow(BlockStmt* blk);
-static void checkErrorHandling(FnSymbol* fn);
+// This is the type for a map that is used when printing out the reason
+// that a function is marked throws. That is relevant for task
+// functions for a coforall, say, where the fact that the body of
+// the coforall throws isn't as relevant to the user as what in
+// particular in the body of the coforall throws.
```

Comment:
The above sentence is quite a puzzle. Let's capitalize the last "what" to help the reader.